ament_index_cpp/include/ament_index_cpp/get_resource.hpp

```diff
@@ -15,10 +15,10 @@
 #ifndef __ament_index_cpp__get_resource__h__
 #define __ament_index_cpp__get_resource__h__
 
-#include <string>
-
 #include <ament_index_cpp/visibility_control.h>
 
+#include <string>
+
```

Comment:
My understanding of this:

https://google-styleguide.googlecode.com/svn/trunk/cppguide.html#Names_and_Order_of_Includes

Is that you would not put `#include <ament_index_cpp/visibility_control.h>` ahead of `#include <string>`. Basically the only thing that goes ahead of system includes are matching header files for the current cpp file. Headers from the same package go last. Again as I see it, based on this example from their document:

```
// file: google-awesome-project/src/foo/internal/fooserver.cc
#include "foo/server/fooserver.h"

#include <sys/types.h>
#include <unistd.h>
#include <hash_map>
#include <vector>

#include "base/basictypes.h"
#include "base/commandlineflags.h"
#include "foo/server/bar.h"
```